tests/e2e/test_operator.py

```diff
@@ -1066,22 +1065,21 @@ def test_019(self):
             },
         )
 
-        with Then("PVC should be re-mounted"):
-        
-            with Then("Non-replicated table should have data"):
-                out = clickhouse.query(chi, sql="select a from t1")
-                assert out == "1"
-                
-            with And("Replicated table should have data"):
-                out = clickhouse.query(chi, sql="select a from t2")
-                assert out == "1"
-            
-    with When("Stop the Installation"):
+    with Then("PVC should be re-mounted"):
```

Comment:
are you sure this changes is required? look like we miss some indent here

Reply:
I melded those tests that were updated in 16.0 - maybe I messed it up
What must be there?

Reply:
I don't know, it depends on your skills during "merging"
if you merge it you should not see changes in tests, cause 0.16.0 contains latest version of tests